tests/lib/tools/suite/system-state/task.yaml

```diff
@@ -0,0 +1,64 @@
+summary: smoke test for the system-state tool
+
+execute: |
+
+    # Check help
+    "$TESTSTOOLS"/system-state | MATCH "usage: is-core"
+    "$TESTSTOOLS"/system-state -h | MATCH "usage: is-core"
+    "$TESTSTOOLS"/system-state --help | MATCH "usage: is-core"
+
+    # check ubuntu core 16
+    if [[ "$SPREAD_SYSTEM" == ubuntu-core-16-* ]]; then
```

Comment:
I'd suggest changing this to a switch statement, it would be easier to read this way IMO

Reply:
done